dhcpv4/nclient4/client.go

```diff
@@ -193,21 +249,19 @@ func (c *Client) Close() error {
 	return err
 }
 
-func isErrClosing(err error) bool {
-	// Unfortunately, the epoll-connection-closed error is internal to the
-	// net library.
-	return strings.Contains(err.Error(), "use of closed network connection")
+func (c *Client) isClosed() bool {
+	return atomic.LoadUint32(&c.closed) != 0
 }
 
 func (c *Client) receiveLoop() {
 	defer c.wg.Done()
 	for {
 		// TODO: Clients can send a "max packet size" option in their
-		// packets, IIRC. Choose a reasonable size and set it.
-		b := make([]byte, maxMessageSize)
+		//       packets, IIRC. Choose a reasonable size and set it.
```

Comment:
Is this indentation on purpose?

Reply:
Yeah (my IDE did not recognize the second line as a "TODO"). But I'll revert it.

Reply:
Fixed.